src/auth/auth.ts

```diff
@@ -75,17 +79,17 @@ const sendSignOutMetrics = async (cause: SignOutCause): Promise<void> => {
     [DEFAULT, () => Events.user.signOut.unspecified]
   );
   const sessionEndTime: number = Date.now();
-  const authStoreState: AuthState = authStore.get();
-  const tokenMetadata: TokenMetadata = authStoreState.authTokenMetadata;
+  const metricStoreState: MetricState = metricStore.get();
+  const tokenMetadata: TokenMetadata = metricStoreState.authTokenMetadata;
```

Comment:
Nit: For consistency, maybe destructure all the fields needed from the metric store instead of renaming this one and then using others directly from `metricStoreState`.